site/src/pages/components/accordion.explainer.mdx

```diff
@@ -81,7 +81,14 @@ Some examples of exclusive accordions on live sites:
 * [Parsons Table product page](https://www.roomandboard.com/catalog/dining-and-kitchen/tables/parsons-tables) at Room and Board (Style/Size/Top/Base accordion on right side of page)
 * [Aaron Dining Chair product page](https://www.potterybarn.com/products/aaron-chair/) at Pottery Barn (Overview / Responsibly Made / Dimensions & Care / Shipping & Returns accordion on right side of page)
 * [Saatva Classic Mattress product page](https://www.saatva.com/mattresses/saatva-classic) (a few pages down, with the first heading "Plush & breathable sleep surface")
-* [Pixel 7 product page](https://store.google.com/product/pixel_7?hl=en-US) on the Google Store (one accordion in each of the sections on Performance, Photography and Video, Protection, and Pixel Portfolio) (though note that this is the type of accordion where exactly one item is open, so it's not exactly a standard exclusive accordion)
+* The "Property Details" section of a [Marriott Hotels hotel page](https://www.marriott.com/en-us/hotels/ispjj-fairfield-inn-new-york-jfk-airport/overview/)
+* ...
+
+Some examples of exclusive accordions where exactly one item is always
+open (discussed below as an exact-exclusive accordion), which is a little
+bit different from the variant that is proposed here:
+* [Pixel 7 product page](https://store.google.com/product/pixel_7?hl=en-US) on the Google Store (one accordion in each of the sections on Performance, Photography and Video, Protection, and Pixel Portfolio)
+* [Zachary's Chicago Pizza FAQ](https://zacharys.com/frequently-asked-questions/)
```

Comment:
In the [exact exclusive](https://open-ui.org/components/accordion.explainer/#accordions-with-exactly-one-item-open) section, it might be good to show roughly what the code looks like to do it. Something like

```css
<style>
details[open]::marker {
  display:none;
}
</style>
<script>
details.addEventListener('toggle',() => details.disabled = details.open);
</script>
```

(I'm sure the above code is flawed somehow, I wrote it quickly.)

If that's true, a) it might be good to add that to the section below, and b) I think it means it's a bit strong to say these examples are "different" from this variant. They just fall into the exact-exclusive category which requires slightly more work.

These are all nits - this LGTM as-is.